pkg/report/report.go

```diff
@@ -50,8 +51,8 @@ type Report struct {
 	Corrupted bool
 	// CorruptedReason contains reason why the report is marked as corrupted.
 	CorruptedReason string
-	// Maintainers is list of maintainer emails (filled in by Symbolize).
-	Maintainers []string
+	// Recipients is a list of RecipientInfo with Email, Real Name, and type.
```

Comment:
The [email RFC](https://tools.ietf.org/html/rfc5322) uses the term "display name", not "real name".  Let's do the same?